apps/chat-e2e/README.md

```diff
@@ -63,14 +63,20 @@ CI report includes screenshots for failed tests.
+| `ENTITY_ARITHMETIC_REQUEST_FOR_API_TESTS` | No       | Dial entities used for simple arithmetic API test                                                                                                                                                          | Should be a valid json corresponding `ArithmeticRequestEntity` interface, e.g. `'[{ "entityId": "gpt-35-turbo", "isSysPromptAllowed": "true" }]'`                                                                                                                          |                       |
+| `ENTITY_PLUS_ADDONS_FOR_API_TESTS`        | No       | Dial entities used for entity+addons API test                                                                                                                                                              | Should be a valid json corresponding `EntityPlusAddonsRequest` interface, e.g. `'[{ "entityId": "gpt-4", "addonIds": ["addon-wolfram"], "request": "plot y = x + 1", "response": "https://www6b3.wolframalpha.com/Calculate/MSP" }]'`                                      |                       |
+| `ENTITY_SIMPLE_REQUEST_FOR_API_TESTS`     | No       | Dial entities used for text request API test, attachment can be expected as a response                                                                                                                     | Should be a valid json corresponding `EntitySimpleRequest` interface, e.g. `'[{ "entityId": "stability.stable-diffusion-xl", "request": "draw smiling emoticon", "isAttachmentResponse": "true"}]'`                                                                        |                       |
+| `ENTITY_PLUS_ATTACHMENT_FOR_API_TESTS`    | No       | Dial entities used for attachment request API test                                                                                                                                                         | Should be a valid json corresponding `EntityPlusAttachmentRequest` interface, e.g. `'[{ "entityId": "gpt-4-vision-preview", "attachmentName": "sun.jpg", "response": "sun" }]'`                                                                                            |                       |
+| `ASSISTANT_PLUS_ADDONS_FOR_API_TESTS`     | No       | Dial entities used for assistant+addons API test                                                                                                                                                           | Should be a valid json corresponding `AssistantPlusAddonsRequest` interface, e.g. `'[{ "assistantId": "test-assistant", "addonIds": ["addonId1", "addonId2"], "assistantModelId": "gpt-4", "request": "assistant request?", "response": "expected assistant response" }]'` |                       |
```

Comment:
Probably add a dedicated section with detailed instructions and examples for implementing new authentication providers? This would make it easier to extend the framework's functionality. **(that is definetely not necessary)**
maybe consider adding a section on how to configure the new API test-related environment variables?

Reply:
the configuration of the new authentication provider and the API test sections have been added to the readme, please review